### PR TITLE
Add docs for npm registry

### PR DESCRIPTION
**Pull request by tianfeng92**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/common-syntax.md [hunk 1]
```diff
@@ -171,10 +171,34 @@ __Example__:
...
+__Example__:
+```yaml
+  registry: https://registry.npmjs.org
+```
+
+⚠️ `registry` can also be private internal npm registry source. It works along with [Sauce Connect](/testrunner-toolkit/running-tests#run-tests-on-sauce-labs-with-sauce-connect) to get access to your private internet from Sauce Cloud.
+
+### `packages`
+
+__Description__: Specifies npm packages which are required.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/common-syntax.md:191]:
```suggestion
__Description__: Specifies npm packages that are required to run tests and should, therefore, be included in the bundle. See [Including Node Dependencies](testrunner-toolkit/configuration#including-node-dependencies).
```

Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/common-syntax.md:191]: Thanks!

File: docs/testrunner-toolkit/configuration/common-syntax.md [hunk 2]
```diff
@@ -171,10 +171,34 @@ __Example__:
...
+__Description__: Specifies npm registry source.
+
+__Type__: *string*
+
+__Example__:
+```yaml
+  registry: https://registry.npmjs.org
+```
+
+⚠️ `registry` can also be private internal npm registry source. It works along with [Sauce Connect](/testrunner-toolkit/running-tests#run-tests-on-sauce-labs-with-sauce-connect) to get access to your private internet from Sauce Cloud.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/common-syntax.md:187]: Hey @tianfeng92  -- I am sorry I didn't catch this before. Have you manually inserted a caution symbol here? Typically, we use the admonition syntax here to put in a warning or caution alert. Is this statement something about which customers should be cautioned? What is the risk level? Thanks and sorry again that I missed this before.

Comment (tianfeng92) [reply docs/testrunner-toolkit/configuration/common-syntax.md:187]: I didn't set this by myself. It was existed previously. I can remove this and add a note section

Comment (sweeneyskirt-sl) [reply docs/testrunner-toolkit/configuration/common-syntax.md]: Huh. When was it added? It is not in the current doc?

File: docs/testrunner-toolkit/configuration/common-syntax.md [hunk 3]
```diff
@@ -171,10 +171,36 @@ __Example__:
...
+__Type__: *string*
+
+__Example__:
+```yaml
+  registry: https://registry.npmjs.org
+```
+
+>
+> **NOTE:** `registry` can also be private internal npm registry source. It works along with [Sauce Connect](/testrunner-toolkit/running-tests#run-tests-on-sauce-labs-with-sauce-connect) to get access to your private internet from Sauce Cloud.
+>
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/common-syntax.md:189]:
```suggestion
__Description__: Specifies the location of the npm registry source. If the registry source is a private address and you are running tests on Sauce Cloud, you can provide access to the registry source using [Sauce Connect](/testrunner-toolkit/running-tests#running-tests-on-sauce-labs-with-sauce-connect).

__Type__: *string*

__Example__:
```yaml
  registry: https://registry.npmjs.org
```

**Review by sweeneyskirt-sl — APPROVED** [review]

**Pull request merged**